tgui/packages/tgui/interfaces/PreferencesMenu/QuirksPage.tsx

```diff
@@ -22,7 +39,14 @@ const QuirkList = (props: {
     }
   ][];
   onClick: (quirkName: string, quirk: Quirk) => void;
+  onCustomizeClick: (e: Event, quirkName: string, quirk: Quirk) => void;
```

Comment:
`onCustomizeClick` does not need to take in the event. The only thing it is used for is to stop propagation. Do that before doing anything else.